doc/policies/conditions.rst

```diff
@@ -0,0 +1,128 @@
+  (see :ref:`useridresolvers`).
+* Then, it retrieves the userinfo attribute given by ``Key``
+* Finally, it uses the ``Comparator`` to compare the contents of the userinfo attribute with the given ``Value``.
+  The result of the comparison determines if the request matches the condition or not.
+
+.. note:: There are situations in which the currently managed user
+    cannot be determined.  If privacyIDEA encounters a policy with ``userinfo``
+    conditions in such a situation, it throws an error and the current request is
+    aborted.
+
+    Likewise, privacyIDEA raises an error if ``Key`` refers to an unknown userinfo
+    attribute, or if the condition definition is invalid due to some other reasons.
+    More detailed information are then written to the logfile.
+
+As an example for a correct and useful ``userinfo`` condition, let us assume
+that you have configured a realm *ldaprealm* with a single LDAP resolver called
+*ldapres*. This resolver is configured to fetch users from a OpenLDAP server,
+with the following attribute mapping:
+
+.. code-block:: json
+
+    { "phone": "telephoneNumber",
+      "mobile": "mobile",
+      "email": "mailPrimaryAddress",
+      "groups": "memberOf",
+      "surname": "sn",
+      "givenname": "givenName" }
+
+You can further define ``groups`` to be a multi-value attribute by setting the
+*Multivalue Attributes* option to ``["groups"]``.
+
+According to this mapping, users of *ldaprealm* will have userinfo entries
+``phone``, ``mobile``, ``email``, ``groups``, ``surname`` and ``givenname``
+which are filled with the respective values from the LDAP directory.
+
+You can now configure a policy that disables the WebUI login for all users in
+the LDAP group ``cn=Restricted Login,cn=groups,dc=test,dc=intranet`` with an
+email address ending in ``@example.com``:
+
+* **Scope**: webui
+* **Action**: ``login_mode=disable``
+* 1) **additional condition** (active):
+    * **Section**: ``userinfo``
+    * **Key**: ``email``
+    * **Comparator**: ``matches``
+    * **Value**: ``.*@example.com``
+  2) **additional condition** (active):
+    * **Section**: ``userinfo``
+    * **Key**: ``groups``
+    * **Comparator:** ``contains``
+    * **Value**: ``cn=Restricted Login,cn=groups,dc=test,dc=intranet``
+
+The policy only takes effect if the user that is trying to log in has a matching
+email address *and* is a member of the specified group. In other words, members
+of the group with an email address ending in ``@privacyidea.org`` will still be
+allowed to log in.
+
+.. note:: Keep in mind that changes in the LDAP directory may not be
+   immediately visible to privacyIDEA due to caching settings (see
+   :ref:`ldap_resolver`).
+
+If the userinfo of the user that is trying to log in does not contain attributes
+``email`` or ``groups`` (due to a resolver misconfiguration, for example), privacyIDEA
+throws an error and the request is aborted.
```

Comment:
Comment and thought:

I know we discussed this and we argued that in such a case, when the value is missing, the decision could not be perfectly made.
But now I think there will be occasions when users will not like this. But I think it is still fine to start this way. Either there will be ways to circumvent this or the admin has to take care, that - if he wants to make decisions on a certain attribute - all users actually have this attribute.

Reply:
Yes, makes sense to keep this in mind! I guess there would be several ways to support "non-strict" condition checking, e.g. by adding more lax comparators (like "matches or does not exist"), or adding a "non-strict" boolean flag to conditions.